### PR TITLE
test: configure testing for frontend code (fehmer)

### PR DESCRIPTION
**Pull request by fehmer**
### Description

Enable unit testing on frontend code.

### CODE REVIEW

File: frontend/jest.config.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+export default {
+  preset: "ts-jest",
+  testEnvironment: "jsdom",
+  transform: tsjPreset.transform,
+  setupFilesAfterEnv: ["<rootDir>/__tests__/setup-tests.ts"],
+  modulePathIgnorePatterns: ["<rootDir>/__tests__/setup-tests.ts"],
+  testRegex: "(/__tests__/.*|(\\.|/)(spec))\\.[jt]sx?$",
+
+  coverageThreshold: {
```
Comment (fehmer) [inline frontend/jest.config.ts:11]: remove or set to zero

File: .eslintrc.json [hunk 2]
```diff
@@ -13,11 +13,12 @@
   },
   "ignorePatterns": [
     "backend/__tests__/**/*",
-    "backend/setup-tests.ts",
```
Comment (Miodec) [inline .eslintrc.json:16]: Why remove this file?

Comment (fehmer) [reply .eslintrc.json:16]: file does not exist in `/backend` and `/backend/__tests__/setup-tests.ts` is ignored by the line above

**Comment by Miodec** [comment]
Thanks

**Pull request merged**